src/main/java/org/thoughtcrime/securesms/ConversationListItem.java

```diff
@@ -165,24 +163,31 @@ public void bind(@NonNull ThreadRecord thread,
     DcContact contact = recipient.getDcContact();
     avatar.setSeenRecently(contact!=null? contact.wasSeenRecently() : false);
 
+    int iconRight = thread.isProtected()? R.drawable.ic_verified : thread.isEmailThread()? R.drawable.ic_outline_email_24 : 0;
     fromView.setCompoundDrawablesWithIntrinsicBounds(
         thread.isMuted()? R.drawable.ic_volume_off_grey600_18dp : 0,
         0,
-        thread.isProtected()? R.drawable.ic_verified : 0,
+        iconRight,
```

Comment:
in general, the letter is a little big too big - maybe better go for 18dp as the volume icon? no blocker, however

Reply:
indeed it was a bit too big, set it to `18sp` as the verified icon and now it is smaller (it looks smaller than the verified icon because the icon has some empty space padding)